lm_eval/models/ibm_watsonx_ai.py

```diff
@@ -93,7 +64,7 @@ def create_from_arg_string(
             from ibm_watsonx_ai.metanames import GenTextParamsMetaNames as GenParams
         except ImportError:
             raise ImportError(
-                "Could not import ibm_watsonx_ai: Please install lm_eval[ibm_watsonx_ai] package."
+                "Could not import ibm_watsonx_ai: Please install ibm_watsonx_ai package."
```

Comment:
I think installing through lm_eval makes more sense, because it may fix the release version.

Reply:
You're right, I've reverted that change.

Reply:
The change was not reverted.